### PR TITLE
feat: upgrade RwTable and add aux

### PR DESCRIPTION
**Pull request by scroll-dev**
to sync with https://github.com/appliedzkp/zkevm-specs/pull/94

### CODE REVIEW

File: zkevm-circuits/src/evm_circuit/util/constraint_builder.rs [hunk 1]
```diff
@@ -503,11 +503,12 @@ impl<'a, F: FieldExt> ConstraintBuilder<'a, F> {
...
             match tag {
-                RwTableTag::TxAccessListAccount => values.swap(2, 3),
+                RwTableTag::TxAccessListAccount => values.swap(3, 4),
                 RwTableTag::TxAccessListStorageSlot => values.swap(3, 4),
-                RwTableTag::Account => values.swap(2, 3),
+                RwTableTag::Account => values.swap(3, 4),
                 RwTableTag::AccountStorage => values.swap(3, 4),
-                RwTableTag::AccountDestructed => values.swap(2, 3),
+                RwTableTag::AccountDestructed => values.swap(3, 4),
+                RwTableTag::TxRefund => values.swap(3, 4),
```
[the remark this answers is not in the document]
Comment (HAOYUatHZ) [reply zkevm-circuits/src/evm_circuit/util/constraint_builder.rs:537]: no problem! I will work on it

[the remark this answers is not in the document]
Comment (HAOYUatHZ) [reply zkevm-circuits/src/evm_circuit/util/constraint_builder.rs]: fixed in [f674018](https://github.com/appliedzkp/zkevm-circuits/pull/285/commits/f6740187a0ff775386eb2241d196cf4d54efed38)

**Review by ed255 — APPROVED** [review]
LGTM!

**Review by icemelon — APPROVED** [review]
LGTM

**Pull request merged**